GitHubRepositories.ps1

```diff
@@ -54,7 +54,11 @@ filter New-GitHubRepository
 
     .PARAMETER Private
         By default, this repository will be created Public.  Specify this to create
-        a private repository.
+        a private repository or use the Visibility parameter.
+
+    .PARAMETER Visibility
+        Specify this to make the repository public, private or internal.  This overrides the
+        private parameter when you use both parameters.
```

Comment:
```suggestion
```suggestion
        Can be public or private. If your organization is associated with an enterprise account
        using GitHub Enterprise Cloud or GitHub Enterprise Server 2.20+, visibility can also be
        internal.

        This will override the Private parameter if also specified in the same command.
```
```